addons/web/static/src/scss/utils.scss

```
@@ -373,7 +373,7 @@
 @mixin print-variable($key, $value) {
     @if $value != null {
         $-type: type-of($value);
-        @if $-type == 'string' {
+        @if $-type == 'string' and str-slice($value, 1, 4) != 'var(' {
```

I'd suggest to use `str-index` that returns  `!= null` when the `var(` substring is found, regardless by its position.
https://sass-lang.com/documentation/modules/string/#index

This way the mixin could handle pure CSS computation, like `calc(var(--font-size) + 1rem)` 

```suggestion
        @if $-type == 'string' and not str-index($value, 'var(') {
```